### PR TITLE
Mint/burn ethereum

### PR DESCRIPTION
**Pull request by MerlinEgalite**

### CODE REVIEW

File: src/MorphoTokenEthereum.sol [hunk 1]
```diff
@@ -16,6 +16,14 @@ contract MorphoTokenEthereum is Token {
...
     string internal constant SYMBOL = "MORPHO";
 
+    /* EVENTS */
+
+    /// @dev Emitted whenever tokens are minted for an account.
+    event Mint(address indexed account, uint256 amount);
+
+    /// @dev Emitted whenever tokens are burned from an account.
+    event Burn(address indexed account, uint256 amount);
+
```
Comment (colin-morpho) [inline src/MorphoTokenEthereum.sol:26]: Shouldn't these be in Token.sol? They are also used in Optimism

Comment (QGarchery) [reply src/MorphoTokenEthereum.sol:26]: done in #45

**Review by QGarchery — APPROVED** [review]
Approving if #45 is merged

**Review by MathisGD — APPROVED** [review]
Let just add a very small test (I can take care of it)

**Review by MerlinEgalite — COMMENTED** [review]
LGTM

**Review by MathisGD — APPROVED** [review]

File: test/MorphoTokenTest.sol
Comment (QGarchery) [inline test/MorphoTokenTest.sol:1]: We could probably rename this file to `MorphoTokenEthereum.sol`

**Review by QGarchery — COMMENTED** [review]
LGTM

**Pull request merged**